lib/src/signed_video_internal.h

```diff
@@ -190,11 +190,13 @@ struct _signed_video_t {
 
   signed_video_authenticity_t *authenticity;  // Pointer to the authenticity report of which results
   // will be written.
+  uint8_t received_gop_hash[MAX_HASH_SIZE];  // Received hash list after decoding SEI data while
```

Comment:
Since you moved it down to the bottom I assume it will be removed in the future; See older review comment. Then I'd like to see a clear separation to the authenticity members above with a TODO to be removed.